### ISSUE TITLE
identify languages doesn't work with Telugu in v0.9

### ISSUE THREAD

**Issue by goru001** [goru001/inltk#57]
identify languages function which uses separate model for identifying the languages hasn't been retrained on Telugu in v0.9.
Need to retrain it to support Telugu.

**Comment by goru001** [comment]
@Shubhamjain27 Will you be able to take this up?

**Comment by lordzuko** [comment]
@goru001 If someone isn't working on this, I can take this up. We can use **pycld2, pycld3** , it identifies all the supported language except: oriya, bengali and sanskrit. 

I have used the same in my own projects and it's also used by **polyglot's** language detection. 
https://github.com/aboSamoor/polyglot/blob/d0d2aa8/polyglot/detect/base.py#L72

What do you think ?

**Comment by goru001** [comment]
@lordzuko That'll be great! Feel free to raise a PR for this.

**Comment by nitkannen** [comment]
@goru001 can I take this issue up if it is still unresolved?

**Comment by goru001** [comment]
@nitkannen Yes sure, this is still unresolved and it'll be great if you can contribute!

**Comment by nitkannen** [comment]
Sure @goru001

**Comment by nitkannen** [comment]
@goru001 can you give me some guidance as to from where I can start to retrain the Telugu model. Any notebooks or scripts used for other languages and data can be really helpful
